### ISSUE TITLE
Related with the caret position during composing Korean characters - IDE 1.6.5

### ISSUE THREAD

**Issue by Myriads** [arduino/Arduino#3223]
Normally(traditionally) the caret is always located at the right side of composing Korean character as like as the attached below picture of IDE 1.6.4.

![arduino_ide 1 6 4](https://cloud.githubusercontent.com/assets/9634845/7791101/3c3375ca-02d5-11e5-9cff-6312e3ba45c4.png)

But in IDE 1.6.5 the caret is located at the left side of composing Korean character:

![arduino_ide 1 6 5](https://cloud.githubusercontent.com/assets/9634845/7791099/37e5ae52-02d5-11e5-8ce5-f4f33806896f.png)

**Comment by ffissore** [comment]
When you type more than one char, do they get printed in the right order?

**Comment by Myriads** [comment]
Yes, characters are inputted in the right order. Just caret is displayed at the left side of composing character.

**Comment by ffissore** [comment]
@Myriads current hourly contains an updated version of the editor component. Can you give it a try and report if the problem persists? http://www.arduino.cc/en/Main/Software#hourly

**Comment by Myriads** [comment]
Caret still has been displayed at the left side of composing Korean character.
